ebill_postfinance/data/transmit.method.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="postfinance_transmit_method" model="transmit.method">
```

Comment:
```suggestion
    <record id="postfinance_transmit_method" model="transmit.method" forcecreate="True">
```

since this xmlid is used in the code, in several places, without failback behavior if missing